ToDoList/Views/Tasks/Doing.cshtml

```diff
@@ -0,0 +1,44 @@
+﻿@model IEnumerable<ToDoList.DTOs.TaskDto>
+@{
+    ViewData["Title"] = "Doing";
+}
+
+
+
+@{
+    foreach (var task in Model)
+    {
+        <div class="container-fluid">
+            <div class="bs-component">
+                <div class="card mb-3">
+                    <h3 class="card-header">Task</h3>
+                    <div class="card-footer text-muted">
+                        Created On @task.CreatedDate.ToShortDateString()
+                    </div>
+                    <div class="card-body">
+                        <h5 class="card-title">@task.Title</h5>
+                        <h6 class="card-subtitle text-muted">Some extra shit</h6>
+                    </div>
+                    <svg xmlns="http://www.w3.org/2000/svg" class="d-block user-select-none" width="100%" height="200" aria-label="Placeholder: Would Images Be Useful?" focusable="false" role="img" preserveAspectRatio="xMidYMid slice" viewBox="0 0 318 180" style="font-size:1.125rem;text-anchor:middle">
+                        <rect width="100%" height="100%" fill="#868e96" data-darkreader-inline-fill="" style="--darkreader-inline-fill:#585f63;"></rect>
+                        <text x="50%" y="50%" fill="#dee2e6" dy=".3em" data-darkreader-inline-fill="" style="--darkreader-inline-fill:#d6d2cd;">Image cap</text>
+                    </svg>
+                    <div class="card-body">
+                        <p class="card-text">@task.Details</p>
+                    </div>
+                    <ul class="list-group list-group-flush">
```

Comment:
There looks to be a lot of front end changes in this sqlite MR. Let's try to get the tickets written for each of the tasks so that the code reviews don't get too large.